.gitignore

```diff
@@ -136,3 +136,4 @@ video_cache_py3.sqlite
 cache.sqlite
 
 chefdata/
+audio_cache.sqlite
```

Comment:
This has been added in develop from your other PR - you are still committing the file itself here as well too, so that needs to be removed from the commit history.